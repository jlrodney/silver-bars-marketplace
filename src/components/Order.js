import React from 'react';

const Order = (props) => (
  <div>
  <p>
    <span>{props.userId} </span>
    <span>{props.quantity} </span>
    <span>{props.price} </span>
    <span>{props.type} </span>
  </p>
  </div>
);

export default Order;
