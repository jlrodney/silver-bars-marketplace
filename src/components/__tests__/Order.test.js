import React from 'react';
import { shallow } from 'enzyme';
import Order from '../Order';

const order = {
    orderId: 'xyz986',
    userId: '1',
    quantity: 22,
    price: 230,
    type: 'BUY'
  }
describe('Expense item', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Order

      key={order.orderId}
      userId={order.userId}
      quantity={order.quantity}
      price={order.price}
      type={order.type}
    />);
    expect(wrapper).toMatchSnapshot();
  });
});
