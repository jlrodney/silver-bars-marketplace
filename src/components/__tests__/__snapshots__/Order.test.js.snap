// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expense item should render correctly 1`] = `
<div>
  <p>
    <span>
      1
       
    </span>
    <span>
      22
       
    </span>
    <span>
      230
       
    </span>
    <span>
      BUY
       
    </span>
  </p>
</div>
`;
